frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -3,11 +3,11 @@ import * as _ from 'lodash';
 import * as classNames from 'classnames';
 import { sortable } from '@patternfly/react-table';
 
-import { getName, getNamespace, getMachineNode } from '@console/shared';
+import { getName, getNamespace, getUid } from '@console/shared';
 import { MachineModel, NodeModel } from '@console/internal/models';
 
 import { MultiListPage, Table, TableRow, TableData } from '@console/internal/components/factory';
-import { ResourceLink, Kebab } from '@console/internal/components/utils';
+import { ResourceLink, Kebab, FirehoseResult } from '@console/internal/components/utils';
```

Comment:
nit: prefer to keep imports alphabetized

Reply:
Done